content/en/infrastructure/process/_index.md

```diff
@@ -134,10 +134,40 @@ Once enabled, use the `AWS Fargate` Containers facet on the [Live Processes page
 
 {{< img src="infrastructure/process/fargate_ecs.png" alt="Processes in AWS Fargate" >}}
 
-For more information about using Datadog Agent with AWS ECS Fargate, see the [ECS Fargate integration documentation][2].
+To enable process monitoring in ECS Fargate, set the `DD_PROCESS_AGENT_PROCESS_COLLECTION_ENABLED` environment variable to `true` in the Datadog Agent container definition within the task definition.
+
+For example:
+
+```json
+{
+    "taskDefinitionArn": "...",
+    "containerDefinitions": [
+        {
+            "name": "datadog-agent",
+            "image": "public.ecr.aws/datadog/agent:7.50.0-rc.1",
+            ...
+            "environment": [
+                {
+                    "name": "DD_PROCESS_AGENT_PROCESS_COLLECTION_ENABLED",
+                    "value": "true"
+                },
+                {
+                    "name": "DD_DOGSTATSD_SOCKET",
+                    "value": "/var/run/datadog/dsd.socket"
+                },
```

Comment:
Let's get of this one. There are other ENV vars we tell users to specify in the main ECS fargate docs, so it may be better not to confuse them and think this is all they need. Keep the focus on process config var.

Reply:
```suggestion
```